contracts/libs/BufferLib.sol

```diff
@@ -2,7 +2,7 @@
 
 pragma solidity >=0.7.0 <0.9.0;
 
-
+import "../exports/WitnetTypes.sol";
```

Comment:
Moving the `Buffer` struct to `WitnetTypes` makes this library depend on `WitnetTypes`. I think that this should be a standalone library, with the `Buffer` struct defined here.

Reply:
Sorry, this is the way to go if the Witnet library is to be dicoupled from the "UsingWitnet" abstract contract, and therefore avoding third party developers to be obliged to link to a (possibly outdated) pre-deployed Witnet library.